chia/server/ws_connection.py

```diff
@@ -310,6 +329,89 @@ async def outbound_handler(self) -> None:
                 self.log.error(f"Exception: {e} with {self.peer_host}")
                 self.log.error(f"Exception Stack: {error_stack}")
 
+    async def _api_call(self, full_message: Message, task_id: bytes32) -> None:
```

Comment:
was this code moved verbatim?

Reply:
Resolving, as this PR has since been approved.